docs/guide/topics/embeds.rst

```diff
@@ -0,0 +1,460 @@
+| |link_embed_google| | |link_embed_discord| | |link_embed_youtube| | |link_embed_github| |
++---------------------+----------------------+----------------------+---------------------+
+
+The examples above are just a few common samples of what you seen often around Discord, these are automatically generated 
+by discord based on the metadata from the site that is being linked but you don't need to remember any of this.
+
+Internally, an embed is represented with a JSON payload by the API. discord.py offers a 
+builder-esque interface over this payload to help make the process more straightforward and intuitive in Python.
+
+Let's take a look at a basic example of using the builder:
+
+.. code-block:: python
+
+    import discord
+
+    embed = discord.Embed(
+        title = "Hello World",
+        description = "This is a description",
+        colour = discord.Colour.blurple()
+    )
+
+And then we can send it to a channel using the ``embed`` keyword-only argument:
+
+.. code-block:: python
+
+    await channel.send(embed=embed)
+
+On Discord, this will look like:
+
+.. image:: /images/guide/topics/embeds/basic_embed.png
+    :scale: 50%
+
+Let's break down what we did.
+
+First, we imported the ``discord`` module. This is required to access the ``Embed`` class.
+
+.. note::
+
+    You can also import the ``Embed`` class directly from the ``discord`` module like so:
+
+    .. code-block:: python
+
+        from discord import Embed
+
+        embed = Embed(...)
+
+Next, we created an instance of the ``Embed`` class. This is the object that will contain the basic fields of our embed.
+
+The ``title`` and ``description`` fields are pretty self-explanatory. The ``colour`` field is a bit more interesting.
+This field is used to set the colour of the left-hand side of the embed. We used the :meth:`discord.Colour.blurple()` classmethod on the 
+:class:`discord.Colour` class that discord.py provides to get the blurple colour. It can also be set to an integer
+representing a hexadecimal colour code like ``0x5865f2`` or ``5793266``.
+
+.. note::
+
+    There is another way to set the basic fields of an embed, you can also set them after construction, like so:
+
+    .. code-block:: python
+
+        embed = discord.Embed()
+        embed.title = "Hello World"
+        embed.description = "This is a description"
+        embed.colour = discord.Colour.blurple()
+
+.. tip::
+    
+    US English spellings can use the respective ``color`` and ``Color`` aliases instead.
+
+.. note::
+    There are two other basic fields that we didn't show here, ``url`` and ``timestamp``. The ``url`` field is used to set the
+    URL that the title of the embed should be masked with. The ``timestamp`` field is used to set the timestamp of the embed. This
+    field takes a ``datetime.datetime`` aware object.
+
+    Try adding these two fields to the embed the same way we did with the other fields and see what happens.
```

Comment:
Having these as separate objects produces a lot of noise that isn't particularly necessary. This can be partially alleviated by moving the first note into the contents itself, although the other two don't particularly fit the flow of the document, so they're probably best as notes.
```suggestion

Instead of passing fields directly to ``Embed``, you can also set basic fields after construction, like so:

.. code-block:: python

    embed = discord.Embed()
    embed.title = "Hello World"
    embed.description = "This is a description"
    embed.colour = discord.Colour.blurple()

.. tip::
    
    US English spellings can use the respective ``color`` and ``Color`` aliases instead.

.. note::
    There are two other basic fields that we didn't show here, ``url`` and ``timestamp``. The ``url`` field is used to set the
    URL that the title of the embed should be masked with. The ``timestamp`` field is used to set the timestamp of the embed. This
    field takes a ``datetime.datetime`` aware object.

    Try adding these two fields to the embed the same way we did with the other fields and see what happens.
```